api/db_queries.py

```diff
@@ -32,6 +32,6 @@
     FOR category IN menu_categories
         FOR collection_key in @collections
             FILTER category["category"] == collection_key
-            RETURN { [category["category"]]: category.categoryname }
+            RETURN { [category["categorynr"]] : { [category["category"]]: category.categoryname } }
```

Comment:
maybe we can just return a sorted array here? Or do we need to have the number as the key?